src/Refactoring-UI/StRefactoringPreviewPresenter.class.st

```diff
@@ -137,7 +137,7 @@ RBRefactoringPreviewPresenter >> initializeLayout [
 ]
```

Comment:
also instead of sending the message `padding:` to each presenter you can set up the spacing of the layout with `aLayout spacing: ` like that you don't need to set the padding each time

Reply:
That's really handy, thanks for the tip!